### PR TITLE
Addressed Checkstyle complaints

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
Another take to make Checkstyle more happy.

### CODE REVIEW

**Comment by BFH-ktt1** [comment]
@MarcinOrlowski , please ask @maehne for review as I'm from now off for 1 week of holidays :-)

**Review by BFH-ktt1 — APPROVED** [review]

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74161.java [hunk 1]
```diff
@@ -44,215 +44,206 @@
...
-                        "A1/QB",
-                        "A0/QA",
-                        "TC/RC0 (Terminal Count)"
-                });
-        super.setInstancePoker(Poker.class);
+  /**
+   * Unique identifier of the tool, used as reference in project files. Do NOT change as it will
+   * prevent project files from loading.
+   *
+   * <p>Identifier value must MUST be unique string among all tools.
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/std/ttl/Ttl74161.java:51]: Why not adding an end tag `</p>` as well?

File: src/main/java/com/cburch/logisim/util/AutoLabel.java [hunk 2]
```diff
@@ -135,28 +135,28 @@ public String getNext(Circuit circ, ComponentFactory me) {
...
-    if (labelBase.containsKey(circ)
-        && currentIndex.containsKey(circ)
-        && useLabelBaseOnly.containsKey(circ)
-        && useUnderscore.containsKey(circ)) active.put(circ, !labelBase.get(circ).isEmpty());
+    if (circ != null) {
+      if (labelBase.containsKey(circ)
+          && currentIndex.containsKey(circ)
+          && useLabelBaseOnly.containsKey(circ)
+          && useUnderscore.containsKey(circ)) active.put(circ, !labelBase.get(circ).isEmpty());
+    }
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/util/AutoLabel.java:152]: Why not merge it into a single `if` statement?

**Review by maehne — COMMENTED** [review]
@MarcinOrlowski: Thanks for these style improvements, which already got merged. Anyway, I noticed some points you may want to consider to address as well.

**Pull request merged**